### PR TITLE
tab error fixed

### PR DESCRIPTION
**Pull request by MarkMed**
Now, don't need to interact directly with tab control, you can change the tab with btns or another extern method (with Btns as @ag-silva did in his test case)

### CODE REVIEW

File: src/components/renders/bootstrap/tab-caption/tab-caption-render.tsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ export class TabCaptionRender implements Renderer {
 
   private clickHandler(event: UIEvent) {
     event.preventDefault();
+    this.component.selected = true;
```
Comment (ncardeli) [inline src/components/renders/bootstrap/tab-caption/tab-caption-render.tsx:57]: I'm worried that, by changing the property here and emitting the `tabSelect` event in the next line, the event will get fired twice when the tab page is clicked. Can you verify it please?

Comment (ncardeli) [reply src/components/renders/bootstrap/tab-caption/tab-caption-render.tsx]: Now we are emitting the `tabSelect` event in two different places. How can we fire it in only one place, so we can have only one source of truth to decide when to emit the event?
We already have a `@Watch` on the `selected` property, so maybe we could take advantage of that.

Comment (MarkMed) [reply src/components/renders/bootstrap/tab-caption/tab-caption-render.tsx]: 🤔

**Comment by MarkMed** [comment]
@ncardeli in the `clickHandler` method should be included the `this.component.selected = true;` line. To change the `select` attribute, and then, the `Watch` decorator will emit the `tabSelect`

**Pull request merged**